src/media/css/site.styl

```diff
@@ -284,7 +284,11 @@ a:active {
     }
     section.full > div {
         padding: 30px 0;
-        width: $max-site-width;
+        max-width: $max-site-width;
+    }
+    .placeholder + .placeholder section.full,
```

Comment:
@ddurst noted that this is a problem basically everywhere. We should probably pitch this PR and come up with a more holistic solution.